opre_ops/ops_site/models.py

```diff
@@ -73,7 +73,7 @@ class CANAmount(models.Model):
     information by fiscal year for a given CAN
     """
     can = models.ForeignKey(CANInfo, on_delete=models.PROTECT)
-    fiscal_year = models.IntegerField(max_length=4)
+    fiscal_year = models.DecimalField(max_digits=4, decimal_places=0)
```

Comment:
[suggestion][blocking]: The resource here suggests the use of validating and bounding the year. We should consider using some form of that, so we can prevent numbers that are not year to be rejected before saving the record.

Reply:
@amymok sounds good, what year do you think we should set as the upper and lower limits??

Reply:
Already posted this in Slack, but I'm not sure this suggestion should be a blocking one. The data model we are implementing here doesn't say anything about a max and a min value for fiscal years. Seems to me that data validation for fiscal years would fall into a different ticket / scope of work.

Reply:
I am ok with this as well.